dts/bindings/mfd/arduino,modulino-buttons-common.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2025 Google, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: Arduino Modulino Buttons (common)
+
```

Comment:
This binding named "-common" that ends up being used in the DT directly instead of being included in another binding kind of confuses/bothers me

Either name arduino,modulino-buttons and then the buttons `arduino,modulino-buttons-buttons`, or just use `i2c-device` directly in the shield overlay?

Reply:
yeah good idea, no point keeping it went for i2c-device, dropped the commit